addons/web/static/src/core/utils/scrolling.js

```diff
@@ -1,5 +1,35 @@
 /** @odoo-module */
 
+/**
+ * Ensure that the pane containing the `element` will be visible in a notebook.
+ *
+ * @param {HTMLElement} element
+ */
+export function displayPane(element) {
```

Comment:
move this to FormRenderer (add a listener to core.bus in on_attach_callback, and call this function when the event is triggered). Don't forget to unbind the handler (in on_detach_callback).